docs/_docs/windows.md

```diff
@@ -12,7 +12,8 @@ For older installations, this page aims to collect some of the general knowledge
 
 *Please note:* You must have [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about?f=255&MSPPError=-2147217396) enabled.
 
-First let's make sure all our packages / repositories are up to date. Open a new Command Prompt instance, and type the following:
+First let's make sure all our packages / repositories are up to date. 
```

Comment:
This change is unnecessary per se as this line break won't be rendered by Kramdown. There is a a lot of inconsistency with respect to this in the entirety of our docs. I prefer handling that in a separate PR.